.env

```diff
@@ -16,9 +16,9 @@ FCREPO_STOMP_PORT=61613
 FCREPO_LOG_LEVEL=INFO
 FCREPO_ACTIVEMQ_CONFIGURATION=classpath:/activemq-queue.xml
 
-COMPACTION_URI=http://example.org/pass/
-COMPACTION_PRELOAD_URI_PASS_STATIC=http://example.org/pass/
-COMPACTION_PRELOAD_FILE_PASS_STATIC=/usr/local/tomcat/lib/context-2.0.jsonld
+COMPACTION_URI=http://oapass.org/ns/pass#
```

Comment:
This is all in fcrepo image, so is unnecessary unless pass-ember devs are making local changes to the context.